lib/plug/conn.ex

```diff
@@ -599,6 +608,9 @@ defmodule Plug.Conn do
   Raises a `Plug.Conn.AlreadySentError` if the connection has already been
   `:sent`.
```

Comment:
"...has already been `:sent` or `:chunked`."?

This way we don't need to repeat the paragraph below.